votingpool/pool.go

```diff
@@ -290,47 +339,75 @@ func (vp *Pool) putSeries(version, seriesID, reqSigs uint32, inRawPubKeys []stri
 //
 // - rawPubKeys has to contain three or more public keys;
 // - reqSigs has to be less or equal than the number of public keys in rawPubKeys.
-func (vp *Pool) CreateSeries(version, seriesID, reqSigs uint32, rawPubKeys []string) error {
-	if series := vp.GetSeries(seriesID); series != nil {
+// FIXME: The minimum allowed value for seriesID should be 1:
```

Comment:
Yeah, it is straightforward but it breaks lots of tests that use seriesID==0, and I didn't want to add even more unrelated changes to this PR, but now it's done :)